concent_api/core/subtask_helpers.py

```diff
@@ -187,3 +188,15 @@ def are_subtask_results_accepted_messages_signed_by_the_same_requestor(subtask_r
         ) for subtask_results_accepted in subtask_results_accepted_list
     )
     return are_all_signed_by_requestor
+
+
+def get_subtask_ids_as_list(client_message: golem_messages.message) -> List[str]:
```

Comment:
`extract_subtask_ids_from_client_message` won't be better name for this function?